api/student.txt

```diff
@@ -0,0 +1,73 @@
+from pydantic import BaseModel, validator
```

Comment:
is it your cheatsheet? :laughing: --> gitignore

Reply:
yes, I wanted to keep the code I made earlier, so I could reference it 🤣  it's getting gitignored now!